stylesheets/_devoxx4kids.scss

```diff
@@ -0,0 +1,20 @@
+.wide-hero {
+	background: cdn("../images/devoxx4kids/devoxx4kids_microsite_hero.jpg") top center no-repeat;
+	background-size:cover;
+	color:white;
+	padding:50px 0;
+	margin-bottom: 40px;
+	border-bottom: 1px solid $grey-2;
+	h3{text-align: left;}
+}
+
+ul{
```

Comment:
I think this might have broken the styling on the primary nav bar on the homepage: http://www-stg.jboss.org/pr/802/build/1731/

Notice the black blocks.